src/plugins/crypto/README.md

```diff
@@ -45,6 +45,7 @@ At the moment the following crypto APIs are supported:
 #ifdef ELEKTRA_CRYPTO_API_BOTAN
 - `libbotan1.10-dev` or `botan-devel`
 #endif
+- gpg command
```

Comment:
a bit more details here: which binaries, which versions, how can it be installed, where does it need to be, so that is found by default, and mention that it is a run-time dependency only